examples/workflows/plot_qc_transforms.py

```diff
@@ -0,0 +1,49 @@
+"""
+Transformations and QC
+----------------------
+
+Built-in transformations using xarray do not
+handle ARM quality control appropriately.  This
```

Comment:
I re-phrase as they are not quality-control aware. It is not necessarily something that xarray should inherently do

Reply:
Good point!  Will update!